src/org/gk/persistence/XMLFileAdaptor.java

```diff
@@ -1122,7 +1122,7 @@ public void clearDeleteRecord(List<Long> dbIDs) {
         // propertyChangeSupport.firePropertyChange("deleteInstance", null, instance);
     }
     
-    public List<?> fetchAllEvents() throws Exception {
+    public List<?> fetchAllEvents() {
```

Comment:
Is there no exception thrown from this method?

Reply:
No, no exception is thrown.  I checked the sequence of methods called in this method's body starting with `fetchInstancesByClass` on line 1833: 

https://github.com/reactome/CuratorTool/pull/14/commits/c72b0ec0f41a31f33c81045eee5f8f72fd4772d8#diff-fbb43a391912c3c0eb5a8816ac6c5cfcR1833

https://github.com/reactome/CuratorTool/pull/14/commits/c72b0ec0f41a31f33c81045eee5f8f72fd4772d8#diff-fbb43a391912c3c0eb5a8816ac6c5cfcR1794

https://github.com/reactome/CuratorTool/pull/14/commits/666e7d62f22701f0fa81d5f3b70f5e186b42e7fc#diff-75d82b36ec4f308fa4a50505e1d758f8R337

and none throw exceptions.  

IntelliJ also confirmed that an exception was not thrown by the method.